CMakeLists.txt

```diff
@@ -132,4 +140,4 @@ if (NOT ZArchive_FOUND)
 	add_subdirectory("dependencies/ZArchive" EXCLUDE_FROM_ALL)
 endif()
 
-add_subdirectory(src)
+add_subdirectory(src)
```

Comment:
It seems that the ending newline was deleted, and Git is unhappy

Reply:
done